tools/pot/openvino/tools/pot/data_loaders/image_loader.py

```diff
@@ -37,4 +40,30 @@ def _read_and_preproc_image(self, img_path):
         if image is None:
             raise Exception('Can not read the image: {}'.format(img_path))
 
-        return prepare_image(image, self.shape[-2:], self._crop_central_fraction)
+        return prepare_image(image, self._layout, self.shape[-2:], self._crop_central_fraction)
+
+    def get_layout(self, input_node):
+        self._layout = None
+        if self._layout is not None:
```

Comment:
this "if" is not going to work since you set it to "None" before.

Reply:
@IndiraSalyahova, why did you just remove an assignment that was previously before the first if? Was it just a bug?

Reply:
@AlexKoff88 this line was not needed, I added it when I tested it and forgot to delete it before pushing.